vocabularies/UI.xml

```diff
@@ -1287,14 +1287,13 @@ The trend is
 
       <Term Name="DisplayTimeZone" Type="Edm.String" AppliesTo="Property">
         <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.RequiresType" String="Edm.DateTimeOffset" />

```

Comment:
You removed the `RequiresType` in order to be able to annotate separate date and time properties.
* How do you declare a (date, time) pair?
* Annotating only a date with an offset makes no sense
* Annotating only a time makes sense (and can lead to something like "10pm on the previous day").

Reply:
Went back to timestamp only